doc/source/tune/key-concepts.rst

```diff
@@ -121,158 +126,237 @@ Each trial is also associated with one instance of a :ref:`Trainable <trainable-
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
     | Trial name           | status   | loc                 |         a |      b |  score | total time (s) |  iter |
     |----------------------+----------+---------------------+-----------+--------+--------+----------------+-------|
-    | MyTrainable_a826033a | RUNNING  | 10.234.98.164:31115 | 0.303706  | 0.0761 | 0.1289 |        7.54952 |    15 |
+    | Trainable_a826033a | RUNNING  | 10.234.98.164:31115 | 0.303706  | 0.0761 | 0.1289 |        7.54952 |    15 |
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
 
 
-You can also easily run 10 trials. Tune automatically :ref:`determines how many trials will run in parallel <tune-parallelism>`.
-
-.. code-block:: python
-
-    tune.run(trainable, config={"a": 2, "b": 4}, num_samples=10)
-
-Finally, you can randomly sample or grid search hyperparameters via Tune's :ref:`search space API <tune-default-search-space>`:
-
-.. code-block:: python
-
-    space = {"x": tune.uniform(0, 1)}
-    tune.run(my_trainable, config=space, num_samples=10)
-
-See more documentation: :ref:`tune-run-ref`.
-
-
-Search spaces
--------------
-
-To optimize your *hyperparameters*, you have to define a *search space*.
-A search space defines valid values for your hyperparameters and can specify
-how these values are sampled (e.g. from a uniform distribution or a normal
-distribution).
+You can also easily run just 10 trials by specifying the number of samples (``num_samples``).
```

Comment:
Should we mention `time_budget_s` here as an alternative? (with `num_samples=-1`) A lot of people don't know about that

Reply:
I didn't know that :D 

Maybe as a side note? I don't want to overload the key concepts section.

Reply:
I think it's important enough that it should be front and center.

Reply:
I added a sentence